my_set/main/templates/main/login.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
Create basic page that will contain shared imports and page structure. Then use "extend" construction to add corresponding content to it